rcl/include/rcl/client.h

```diff
@@ -339,6 +339,11 @@ RCL_WARN_UNUSED
 const rcl_client_options_t *
 rcl_client_get_options(const rcl_client_t * client);
 
+///  Non-safe get_options function.
+/* *INDENT-OFF* */
+#define _client_get_options(client) &client->impl->options;
```

Comment:
Does this need to be in the header? It would be better to put it in the `.c` file if possible. It shouldn't be part of the public API if possible (even if it has the leading `_`).